recipes/xrandr.mdwn

```diff
@@ -0,0 +1,16 @@
+# Multiple Monitor Helper
+
+This page is intended to show how to user `xrandr.lua` script to help the way you deal with multiple monitors.
+Using this snippet you can set a keybinding where you swap to all `(Number of Monitors)!` screen setups.
+
+The process of setting up this script goes as follow:
+
+1. Create a file called `xrandr.lua` in your file system (preferably in awesome's folder) with the [script](xrandr.lua) content.
+
+2. Import the script in your `rc.lua`
+
+          local xrandr = require("/path/to/file/xrandr")
+
+3. Create a keybinding to the script in the appropriated globalkeys table
+
+           awful.key({}, "Some Key", function() xrandr.xrandr() end)
```

Comment:
It might make sense to explain what this key binding will do. From reading the code I guess:

Pressing this key binding will open a popup with a possible screen arrangement. Pressing the key again will replace this popup with another possibility, eventually arriving at "keep the current configuration". If the key is not pressed again within four seconds, the configuration described in the current popup is applied.